schematic/store/synapse.py

```diff
@@ -2281,12 +2284,9 @@ def _update_table_uuid_column(self, sg: SchemaGenerator,) -> None:
                         "maximumListLength": 1,
                     }
 
-                    # Send POST /column request to define new column and get new column ID
-                    newColResponse = self.synStore.send_api_request(
-                        request_type = "restPOST",
-                        uri = "https://repo-prod.prod.sagebase.org/repo/v1/column",
-                        body = columnModelDict,
-                    )
+                    new_col = Column(name='Id', columnType='STRING', maximumSize=64, maximumListLength=1)
+                    newColResponse = self.synStore.syn.store(new_col)
```

Comment:
Nit: Why camel case here?

Reply:
hmm I copied Gianna's variable name in her original PR. But I think our code base is mostly using snake case. I should just switch that back to snake case.